File: C#/BluffinMuffin.Protocol/DisconnectCommand.cs
using BluffinMuffin.Protocol.Enums;

namespace BluffinMuffin.Protocol
{
    /// <summary>
    /// This command is sent to inform the other end of the Tcp Connection that the communication should end
    /// </summary>
    public class DisconnectCommand : AbstractCommand
    {
        /// <summary>
        /// 
        /// </summary>
        public override BluffinCommandEnum CommandType
        {
            get { return BluffinCommandEnum.General; }
        }
    }
}

File: Python/bluffinmuffin/protocol/disconnect_command.py
from bluffinmuffin.protocol.interfaces import AbstractCommand
from bluffinmuffin.protocol.enums import BluffinCommandEnum


class DisconnectCommand(AbstractCommand):

    def __init__(self):
        super().__init__(BluffinCommandEnum.General)

    @classmethod
    def decode(cls, obj):
        return cls()
